environment_setup/tools/macos_fw.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/src/WORKSPACE b/src/WORKSPACE
+index a18004d..cfaf0d9 100644
+--- a/src/WORKSPACE
++++ b/src/WORKSPACE
+@@ -97,10 +97,10 @@ http_archive(
+ http_archive(
+     name = "arm_developer_gcc",
+     build_file = "@//external:arm_gcc.BUILD",
+-    sha256 = "b50b02b0a16e5aad8620e9d7c31110ef285c1dde28980b1a9448b764d77d8f92",
+-    strip_prefix = "gcc-arm-none-eabi-8-2019-q3-update",
++    sha256 = "1249f860d4155d9c3ba8f30c19e7a88c5047923cea17e0d08e633f12408f01f0",
++    strip_prefix = "gcc-arm-none-eabi-9-2019-q4-major",
+     urls = [
+-        "https://developer.arm.com/-/media/Files/downloads/gnu-rm/8-2019q3/RC1.1/gcc-arm-none-eabi-8-2019-q3-update-linux.tar.bz2",
++        "https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/RC2.1/gcc-arm-none-eabi-9-2019-q4-major-mac.tar.bz2",
+     ],
```

Comment:
Hmm, I wonder if there's a way for move this down into a bazel rules file where we can actually switch on OS?

Reply:
Yeah I think we discussed this last week, Bazel doesn't allow `select` in the WORKSPACE file other this would be trivial.
Since all the wrapper scripts for the toolchain are hardcoded to use `arm_developer_gcc`, this isn't a trivial change. I don't have the Bazel-fu to know of a reasonable solution to this, so...

Reply:
Hmm. We could maybe come up with something, but I don't want to make any compromises to the simplicity of the build system in order to support MacOS.... this is probably worth an in-person discussion.